### PR TITLE
What's New in DevTools 130

### PR DESCRIPTION
**Pull request by leahmsft**
Rendered articles for review:
* **What's New in DevTools (Microsoft Edge 130)**
   * https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130?branch=pr-en-us-3290
   * [Planned live](https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130)
* **Emulate mobile devices (Device Emulation)**
   * https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/device-mode/?branch=pr-en-us-3290
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/device-mode/)
   * Removed over-prominent Limitations h2 heading.
      * Moved its content up with intro sentence that got lost between the inline TOCs.
      * Framed the content as intro description rather than limitations.
* **Experimental features in Microsoft Edge DevTools**
   * https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/experimental-features/?branch=pr-en-us-3290
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/experimental-features/)
   * Not updated.  Non-blocking; nice to do.

AB#54505536

### CODE REVIEW

**Review by mikehoffms — APPROVED** [review]
LGTM - Writer/Editor

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+![Network Console collection and environment](./devtools-130-images/create-collection-environment.png)
+
+See also:
+* [Compose and send web API requests using the Network Console tool](../../../network-console/network-console-tool.md)
+
+
+<!-- ====================================================================== -->
+## Bug fixes for icons
+
+<!-- Subtitle: todo -->
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md:60]: add subtitle for Bug fixes for icons
@leahmsft

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+   * [Recommendations in live metrics](https://developer.chrome.com/blog/new-in-devtools-130#live-metrics-recommendations)
+   * [Navigate breadcrumbs](https://developer.chrome.com/blog/new-in-devtools-130#breadcrumbs)
+* [Memory panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#memory)
+   * [New 'Detached elements' profile](https://developer.chrome.com/blog/new-in-devtools-130#detached-elements)
+   * [Improved naming of plain JS objects](https://developer.chrome.com/blog/new-in-devtools-130#object-names)
+* [Turn off dynamic theming](https://developer.chrome.com/blog/new-in-devtools-130#dynamic-theming)
+* [Chrome Experiment: Process sharing](https://developer.chrome.com/blog/new-in-devtools-130#process-sharing)
+* [Lighthouse 12.2.1](https://developer.chrome.com/blog/new-in-devtools-130#lighthouse)
+* [Miscellaneous highlights](https://developer.chrome.com/blog/new-in-devtools-130#misc)
+<!-- todo: maybe remove some links -->
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md:144]: trim chromium links 
@leahmsft

**Review by leahmsft — COMMENTED** [review]
LGTM - pm

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+See also:
+* [Throttle the network and CPU](../../../device-mode/index.md#throttle-the-network-and-cpu) in _Emulate mobile devices (Device Emulation)_.
+
+
+<!-- ====================================================================== -->
+## Console's Warnings count button has increased contrast
+
+<!-- Subtitle: The warning icon and expand button on warning messages in the Console now meet the standard luminosity ratio of 3:1.-->
+
+In the **Console** tool, the Warning count expander button on warning messages in the **Console** previously didn't meet the standard luminosity ratio of 3:1:
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md:109]:
```suggestion
In the **Console** tool, the Warning count expander button on warning messages in the **Console** previously didn't meet the standard luminosity ratio of 3:1.
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+   * [HAR exports now exclude sensitive data by default](https://developer.chrome.com/blog/new-in-devtools-130#har)
+* [Elements panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#elements)
+   * [Autocomplete values for text-emphasis-* properties](https://developer.chrome.com/blog/new-in-devtools-130#autocomplete)
+   * [Scroll overflows marked with a badge](https://developer.chrome.com/blog/new-in-devtools-130#scroll-badge)
+   * [Bare declarations after nested rules don't "shift up"](https://developer.chrome.com/blog/new-in-devtools-130#css-nesting)
+* [Performance panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#perf)
+   * [Recommendations in live metrics](https://developer.chrome.com/blog/new-in-devtools-130#live-metrics-recommendations)
+   * [Navigate breadcrumbs](https://developer.chrome.com/blog/new-in-devtools-130#breadcrumbs)
+* [Memory panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#memory)
+   * [New 'Detached elements' profile](https://developer.chrome.com/blog/new-in-devtools-130#detached-elements)
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md:138]: Well, we contributed this upstream, and we're already covering this in the **Detached Elements profiling type in Memory tool** section. So we don't need to repeat it here.

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+   * [Improved naming of plain JS objects](https://developer.chrome.com/blog/new-in-devtools-130#object-names)
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md:139]: Same thing here. I believe this is covered by the **Improved names of JavaScript objects in heap snapshots** section.

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md:138]: removed chromium links:
```
* [Memory panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#memory)
   * [New 'Detached elements' profile](https://developer.chrome.com/blog/new-in-devtools-130#detached-elements)
   * [Improved naming of plain JS objects](https://developer.chrome.com/blog/new-in-devtools-130#object-names)
```

* [Memory panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#memory)
   * [New 'Detached elements' profile](https://developer.chrome.com/blog/new-in-devtools-130#detached-elements)
   * [Improved naming of plain JS objects](https://developer.chrome.com/blog/new-in-devtools-130#object-names)

@leahmsft

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md (hunk 5 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md:139]: removed chromium links:
```
* [Memory panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#memory)
   * [New 'Detached elements' profile](https://developer.chrome.com/blog/new-in-devtools-130#detached-elements)
   * [Improved naming of plain JS objects](https://developer.chrome.com/blog/new-in-devtools-130#object-names)
```

* [Memory panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#memory)
   * [New 'Detached elements' profile](https://developer.chrome.com/blog/new-in-devtools-130#detached-elements)
   * [Improved naming of plain JS objects](https://developer.chrome.com/blog/new-in-devtools-130#object-names)

@leahmsft

**Review by captainbrosset — APPROVED** [review]

**Pull request merged**